docs/maya/muscle.md

```diff
@@ -87,9 +87,11 @@ To create an AdnMuscle, follow these steps:
 | Name | Type | Default | Animatable | Description |
 | :--- | :--- | :------ | :--------- | :---------- |
 | **Attenuation Matrix**           | Matrix | Identity | ✓ | Transformation matrix to drive the attenuation. |
-| **Attachment Matrix**            | Matrix | Identity | ✓ | List of attachment matrices  (from a compound attribute) used for setting up attachments. |
+| **Attachment Matrix**            | Matrix | Identity | ✓ | List of transform attachment matrices  (from a compound attribute) used for setting up transform attachments. |
 | **Slide On Segment Root Matrix** | Matrix | Identity | ✓ | List of root matrices (from a compound attribute) used for setting up segments of slide on segment constraints. |
 | **Slide On Segment Tip Matrix**  | Matrix | Identity | ✓ | List of tip matrices (from a compound attribute) used for setting up segments of slide on segment constraints. |
+| **Target World Mesh**            | Mesh   |          | ✗ | List of geometry attachment meshes (from a compound attribute) used for setting up attachments to geometries. |
```

Comment:
Question. Why is this attribute marked as non animatable?

Reply:
I missed this one probably from copy pasting.